packages/cashier/src/constants/routes-config.ts

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
 import { routes, moduleLoader } from '@deriv/shared';
+import { Redirect } from 'react-router-dom';
 import { localize } from '@deriv/translations';
 import { Cashier } from '../containers';
 import { AccountTransfer, Deposit, OnRamp, P2PCashier, PaymentAgent, PaymentAgentTransfer, Withdrawal } from '../pages';
 
+export type TRoute = {
```

Comment:
As these types are being used in other files, isn't it better to create a file in the `types` folder named, for example, `route.types.ts` and use it across all files?

Reply:
sorted